src/coprocessor_v2/plugin_registry.rs

```diff
@@ -306,12 +315,22 @@ impl PluginRegistryInner {
         &mut self,
         filename: P,
     ) -> Result<String, PluginLoadingError> {
+        if self.library_paths.contains(filename.as_ref()) {
+            let err = Err(PluginLoadingError::ReloadError {
+                path: filename.as_ref().to_owned(),
+            });
+            let filename = filename.as_ref().to_string_lossy();
```

Comment:
`OsStr` should have implemented `Debug`, so we do not need to convert that to `String` here. Does the compiler complains if we remove `.to_string_lossy()`?

Reply:
Yes `to_string_lossy()` is optional. But this line is copied from [the same function](https://github.com/tikv/tikv/blob/ef5a703dc671e9eb0583f90be7f8b5b1552b6ab2/src/coprocessor_v2/plugin_registry.rs#L328). Is there a reason not to be consistent?